courses/fundamentals_of_ada/030_basic_types.rst

```diff
@@ -470,71 +341,6 @@ Predefined Modular Types
    type Unsigned_8 is mod 2 ** 8;
    type Unsigned_16 is mod 2 ** 16;
 
-------------------------
```

Comment:
Keep the Shift/Rotate page and Bit-Oriented examples page - I think this important as to the benefits of mod.
(You can leave the "implicit" and  "user-defined" slides only in Advanced)